SofieCompileModels_PyTorch.vcxproj fails to compile on pytest version 6.2.5

### Describe the bug
Hi All,

I am trying to build ROOT project as well as tests, but there is an error occurred when building, and it maybe related to pytest version. When I use same build steps but different pytest version, ROOT project build success with pytest version 6.1.1, but SofieCompileModels_PyTorch.vcxproj fails to compile with pytest version 6.2.5, the error as belows. I have checked if pytest version is 6.1.1, there is no pymva folder generated after cmake commands, and for pytest version 6.2.5, pymva folder generated after cmake commands. Could you please help have a look at this issue? Thanks a lot!
![image](https://user-images.githubusercontent.com/17844901/147636210-34c8102b-8380-4b92-828d-097a6af7829d.png)


C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: The command "setlocal [F:\gitP\root-project\root\build_amd64\tmva\pymva\test\SofieCompileModels_PyTorch.vcxproj]
       C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: .\emitFromPyTorch [F:\gitP\root-project\root\build_amd64\tmva\pymva\test\SofieCompileModels_PyTorch.vcxproj]
       C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [F:\gitP\root-project\root\build_amd64\tmva\pymva\test\SofieCompileModels_PyTorch.vcxproj]
       C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: :cmEnd [F:\gitP\root-project\root\build_amd64\tmva\pymva\test\SofieCompileModels_PyTorch.vcxproj]
       C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone [F:\gitP\root-project\root\build_amd64\tmva\pymva\test\SofieCompileModels_PyTorch.vcxproj]
       C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: :cmErrorLevel [F:\gitP\root-project\root\build_amd64\tmva\pymva\test\SofieCompileModels_PyTorch.vcxproj]
       C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: exit /b %1 [F:\gitP\root-project\root\build_amd64\tmva\pymva\test\SofieCompileModels_PyTorch.vcxproj]
       C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: :cmDone [F:\gitP\root-project\root\build_amd64\tmva\pymva\test\SofieCompileModels_PyTorch.vcxproj]
       C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: if %errorlevel% neq 0 goto :VCEnd [F:\gitP\root-project\root\build_amd64\tmva\pymva\test\SofieCompileModels_PyTorch.vcxproj]
       C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: :VCEnd" exited with code 9009. [F:\gitP\root-project\root\build_amd64\tmva\pymva\test\SofieCompileModels_PyTorch.vcxproj]

### Expected behavior
Project build successfully.

### To Reproduce
1. Open a x64 Native Tools Command Prompt for VS 2019
2. git clone --branch master https://github.com/root-project/root.git
3. cd root
4. mkdir build_amd64 && cd build_amd64
5. pip install pytest
5. cmake -G "Visual Studio 16 2019" -A x64 -DCMAKE_SYSTEM_VERSION=10.0.18362.0 -Droofit=off -Dtesting=ON -Droottest=ON ..
6. cmake --build . --config Release

### Setup
1. Operating system: Windows 10
2. How you obtained ROOT, built it myself.

--- a/tmva/pymva/test/CMakeLists.txt
+++ b/tmva/pymva/test/CMakeLists.txt
@@ -59,19 +59,26 @@ if(PY_TORCH_FOUND)
    set_target_properties(emitFromPyTorch PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
    add_custom_target(SofieCompileModels_PyTorch)
    add_dependencies(SofieCompileModels_PyTorch emitFromPyTorch)
-   add_custom_command(TARGET SofieCompileModels_PyTorch POST_BUILD
-		COMMAND ./emitFromPyTorch
-		USES_TERMINAL
-	)
-   ROOT_ADD_GTEST(TestRModelParserPyTorch TestRModelParserPyTorch.C
-    LIBRARIES
-    ROOTTMVASofie
-    blas
-    ${PYTHON_LIBRARIES}
-    INCLUDE_DIRS
-    ${CMAKE_CURRENT_BINARY_DIR}
-   )
-   add_dependencies(TestRModelParserPyTorch SofieCompileModels_PyTorch)
+   if(MSVC)
+      add_custom_command(TARGET SofieCompileModels_PyTorch POST_BUILD
+         COMMAND $<CONFIG>/emitFromPyTorch.exe
+         USES_TERMINAL
+      )
+   else())
+      add_custom_command(TARGET SofieCompileModels_PyTorch POST_BUILD
+         COMMAND ./emitFromPyTorch
+         USES_TERMINAL
+      )
+      ROOT_ADD_GTEST(TestRModelParserPyTorch TestRModelParserPyTorch.C
+         LIBRARIES
+         ROOTTMVASofie
+         blas
+         ${PYTHON_LIBRARIES}
+         INCLUDE_DIRS
+         ${CMAKE_CURRENT_BINARY_DIR}
+      )
+      add_dependencies(TestRModelParserPyTorch SofieCompileModels_PyTorch)
+   endif()
 endif(PY_TORCH_FOUND)
 
 if((PY_KERAS_FOUND AND PY_THEANO_FOUND) OR (PY_KERAS_FOUND AND PY_TENSORFLOW_FOUND))

--- a/tmva/pymva/test/CMakeLists.txt
+++ b/tmva/pymva/test/CMakeLists.txt
@@ -64,7 +64,7 @@ if(PY_TORCH_FOUND)
          COMMAND $<CONFIG>/emitFromPyTorch.exe
          USES_TERMINAL
       )
-   else())
+   else()
       add_custom_command(TARGET SofieCompileModels_PyTorch POST_BUILD
          COMMAND ./emitFromPyTorch
          USES_TERMINAL

